pkg/agent/datamodel/mariner_v2_kata_gen2_tl_sig_version.json

```
@@ -0,0 +1,4 @@
+{
+    "ostype": "CBLMarinerV2KataGen2TLSIGImageVersion",
+    "version": "2023.01.25"
```

```suggestion
    "version": "0000.00.00"
```

to avoid accidentally using 2023.01.25 which had kernel issues and we hotfixed

---

pinned to 2022.12.15 which is a good version.